### PR TITLE
Add list and character component

### PR DESCRIPTION
**Pull request by buentellopc**
We will retrieve the data in order to fulfill the List and Character components.

Using the following API: https://rickandmortyapi.com/documentation/#rest

### CODE REVIEW

File: src/components/List.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Character from "./Character";
+
+function List() {
+  return;
+  <div>
+    <h2>Characters</h2>;
+    <Character />
+    <Character />
+    <Character />
```
Comment (frxncismor) [inline src/components/List.js]: We don't need this third character line, can you please delete it?
```suggestion
  
```

File: src/components/Character.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.card {
+  /* border: 1px solid red; */
```
Comment (frxncismor) [inline src/components/Character.css]: Please delete the code you aren't using anymore in order to keep code clean
```suggestion
  
```

File: src/App.js [hunk 3]
```diff
@@ -1,23 +1,24 @@
 import logo from "./logo.svg";
 import "./App.css";
+import List from "./components/List";
+import Search from "./components/Search";
+import ParentSearch from "./components/ParentSearch";
```
Comment (buentellopc) [inline src/App.js]:
```suggestion

```

**Review by frxncismor — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 4]
```diff
@@ -1,23 +1,25 @@
...
-        >
-          HI mom!
-        </a>
+    <div>
+      <header className="header">
+        <h1 className="header__title">Rick and Morty</h1>
+        <Search pattern={pattern} patternHandler={patternHandler} />
       </header>
+      <List pattern={pattern} />
+      {/* <InfiniteScroll /> */}
```
Comment (frxncismor) [inline src/App.js]:
```suggestion

```

**Review by frxncismor — APPROVED** [review]

**Pull request merged**